### ISSUE TITLE
[Consistency] Make sure all xxxForSequenceClassification models support problem_type

### ISSUE THREAD

**Issue by NielsRogge** [huggingface/transformers#13370]
A while ago (#11012), an additional attribute called `problem_type` has been added to `xxxForSequenceClassification` models, which you can set to "multi_label_classification", "single_label_classification" or "regression" to fine-tune `xxxForSequenceClassification` models for the respective problem. This makes sure the appropriate loss function is used.

This is great, however, 3 things need to be improved in my opinion:

* this is causing a bit of an inconsistency as it's not implemented in all models which have an `xxxForSequenceClassification` head model.
* it's also not included in any of the CookieCutter templates, which are used to add new models.
* this is not documented anywhere.

cc @abhishekkrthakur

**Comment by abhishekkrthakur** [comment]
Pretty sure it is documented :) 
https://huggingface.co/transformers/main_classes/configuration.html

Agree that its not in cookie-cutter and its not implemented for all models (it cannot be implemented for all). :)

![image](https://user-images.githubusercontent.com/1183441/131682585-2d0b4a83-0885-4536-81bd-e6b46a78eecd.png)
